app/templates/components/nav-bar.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="mobile hidden row">
   <div class="ui menu">
     {{#link-to 'index' class='header item' activeClass=''}}
-      <h3>{{settings.appName}}</h3>
+      <h3>{{config.appName}}</h3>
```

Comment:
Why are you changing `Mobile Hidden Row`, Change the name in `Mobile Row` to `settings.appName` . That's it.
@iamareebjamal Talking about the global injection of settings that can be handled later once this PR is done.